### DISCUSSION TITLE
[Enhancement] [select-v2] select-v2 focus/blur method

### DISCUSSION THREAD

**Discussion by YujiaCheng1996** [element-plus/element-plus#12006] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select-v2`

## Description

Select-v2 should be able to focus/blur manually.
希望提供能手动触发Select-v2的focus/blur方法。

### Additional comments

_(empty)_

**Comment by kooriookami** [comment]
Added in 2.5.0
